delivery_postlogistics/postlogistics/web_service.py

```diff
@@ -14,7 +14,7 @@
 import requests
 from PIL import Image
 
-from odoo import _, exceptions
+from odoo import _
```

Comment:
```suggestion
```
This can be removed since `*.env._` is used instead

Reply:
This is the webservice implementation which does not inherits from Models ^^